src/callbacks.jl

```diff
@@ -0,0 +1,77 @@
+"""
+    Callbacks(; lazy = nothing, heuristic = nothing)
+
+Set MIP-based callbacks. Two basic types are offered: `lazy` and `heuristic`.
+
+### The lazy callback
+
+The `lazy` callback can be used to add *lazy* constraints which are added
+on-demand at integer nodes in the branch and bound tree.
+
+Points to consider are:
+
+- The lazy constraint callback *may* be called when the solver has an integer
+  primal solution.
+- You can only access the primal solution through `VariablePrimal`. For example,
+  `ConstraintDual` etc will not work.
+- The optimal solution will satisfy all lazy constraints that could *possibly*
```

Comment:
What does this mean? What are the constraints that could possibly have been added and how do they differ from the constraints actually returned?

Reply:
Poor wording on my part. It should just say all constraints that have been added. I'll hold off changing until we have a consensus on the larger issue of whether we want these in the first place.